### PR TITLE
Add Relm4 to other crates

### PR DESCRIPTION
**Pull request by AaronErhardt**

### CODE REVIEW

File: _posts/2022-01-14-new-release.md [hunk 1]
```diff
@@ -221,6 +221,8 @@ Other than the classic [gtk-rs][gtk-rs] crates, we have also worked on ensuring
  - [libshumate-rs](https://world.pages.gitlab.gnome.org/Rust/libshumate-rs/): released an alpha for the GTK4 Map widgetery library
  - [sourceview4-rs](https://world.pages.gitlab.gnome.org/Rust/sourceview4-rs/)/[libhandy-rs](https://world.pages.gitlab.gnome.org/Rust/libhandy-rs/)/[libdazzle-rs](https://world.pages.gitlab.gnome.org/Rust/libdazzle-rs/)/[gspell-rs](https://world.pages.gitlab.gnome.org/Rust/gspell-rs/)/[poppler-rs](https://world.pages.gitlab.gnome.org/Rust/poppler-rs/)
  - [libflatpak-rs](https://world.pages.gitlab.gnome.org/Rust/libflatpak-rs/): first release of the libflatpak bindings
+
```
Comment (GuillaumeGomez) [inline _posts/2022-01-14-new-release.md:224]: Please follow the existing list format. No need to add all the rest too, people can see it when they go on the repository.

Comment (AaronErhardt) [reply _posts/2022-01-14-new-release.md:224]: But Relm4 isn't exactly part of GNOME World, so if it's added to the list, the list heading needs to be updated as well.

Comment (GuillaumeGomez) [reply _posts/2022-01-14-new-release.md:224]: Then update the first sentence of the section (which is named "other crates").

Comment (sdroege) [reply _posts/2022-01-14-new-release.md:224]: > But Relm4 isn't exactly part of GNOME World

Neither is GStreamer

**Comment by AaronErhardt** [comment]
Not related to this issue, but I found it really hard to follow the release discussions and in general internal topics in the gtk-rs matrix room. There are now so many people active in this room that sometimes questions are overlooked and often internal discussions are interrupted by other messages, which I believe even led to misunderstandings in the past. Maybe it'd be better to have two rooms, one for internal discussions and one for questions. That'd at least make my life easier.

**Review by sdroege — APPROVED** [review]

**Comment by sdroege** [comment]
> Not related to this issue, but I found it really hard to follow the release discussions and in general internal topics in the gtk-rs matrix room. There are now so many people active in this room that sometimes questions are overlooked and often internal discussions are interrupted by other messages, which I believe even led to misunderstandings in the past. Maybe it'd be better to have two rooms, one for internal discussions and one for questions. That'd at least make my life easier.

Not sure how that's going to work out in practice. How do you properly distinguish what is a question and what's an internal discussion? :) But I agree that the Matrix room became a bit noisy sometimes.

@bilelmoussaoui @GuillaumeGomez maybe you have an idea?

**Comment by GuillaumeGomez** [comment]
We have an issue open for that. If they need to discuss something, they can always open another issue and say it's linked to the release?

**Comment by bilelmoussaoui** [comment]
Tbh, the growth of the matrix room is okayish, I mean it is kind of expected a ton of new gtk apps are mostly written in Rust nowadays so. I think having a room for internal discussions means one more room to look at and follow for whatever on top of the various others things to do, so I would rather not have that. If there is something very important to discuss it could go through an issue :)
